csrc/u8g2_d_memory.c

```diff
@@ -3,13 +3,19 @@
 
 #include "u8g2.h"
 
+#ifdef U8G2_USE_ONE_STATIC_BUFF
+static uint8_t buf[1024];
+#endif // U8G2_USE_ONE_STATIC_BUFF
+
 uint8_t *u8g2_m_16_4_1(uint8_t *page_cnt)
 {
   #ifdef U8G2_USE_DYNAMIC_ALLOC
   *page_cnt = 1;
   return 0;
   #else
+  #ifndef U8G2_USE_ONE_STATIC_BUFF
```

Comment:
I do not see the need for U8G2_USE_ONE_STATIC_BUF.
U8g2 will crash if U8G2_USE_DYNAMIC_ALLOC is not defined and U8G2_USE_ONE_STATIC_BUFF is defined.
Moreover u8g2_d_memory.c is generated code. Changes to this file directly does not make sense.

Reply:
ok，already removed U8G2_USE_ONE_STATIC_BUFF